src/__test__/cursor.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-env node, jest */
+const { cursor, asyncCursor, asyncToArray } = require('iter-tools')
+
+describe('cursor', function () {
+  it('frames iterable', function () {
+    const iter = cursor(3, [1, 2, 3, 4, 5])
+    expect(Array.from(iter)).toEqual([
+      [1, 2, 3],
+      [2, 3, 4],
+      [3, 4, 5]
+    ])
+  })
+
+  it('frames iterable (cursor equal to the sequence)', function () {
+    const iter = cursor(5, [1, 2, 3, 4, 5])
+    expect(Array.from(iter)).toEqual([
+      [1, 2, 3, 4, 5]
+    ])
+  })
+
+  it.only('frames iterable (cursor bigger than the sequence)', function () {
+    const iter = cursor(6, [1, 2, 3, 4, 5])
+    expect(Array.from(iter)).toEqual([
+      [undefined, 1, 2, 3, 4, 5]
```

Comment:
Again I think it should just be a smaller frame. Or, if you're going to fill values, I think you should allow the user to specify what to fill them with.

Reply:
I don't think that using a smaller frame is an option. I imagine this to be used like this:
```js
for (const [previous, current] of cursor(2, iterable)) {
  ...
}
```
And you can easily manage the undefined value case by case. Just like we do in zip-longest

Reply:
I agree that that is a fairly compelling use case, but if so then you've already hurt it by removing the leading undefineds (at my request). 

I guess I'm starting to think that there are really two methods here: leading cursor and trailing cursor. And I think in both cases current needs to be every item in the list during a cursor run. But that way the pattern would be:

```js
for (const [previous, current] of trailingCursor(2, iterable)) {
  ...
}
```
and 
```js
for (const [current, next] of leadingCursor(2, iterable)) {
  ...
}
```

Reply:
I like the idea. I implemented an option "trailing" to alternate between the 2 variations